src/hooks/v2v3/transactor/LaunchProjectWithNftsTx.ts

```diff
@@ -1,30 +1,34 @@
 import { getAddress } from '@ethersproject/address'
+import { BigNumber } from '@ethersproject/bignumber'
```

Comment:
The diff in this file is the core update. Makes data structure modifications, to give the new nft rewards tx what it expects.